lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -109,14 +109,14 @@ confederation_id:
     config_set_append: '<state> confederation identifier <id>'
 
 confederation_peers:
-  # TODO: Supported in XR but broken as identifier is nvgened on a seperate line.
   default_value: ""
```

Comment:
The issue as I understand it is that in XR this is:
```
bgp confederation peers
  1
  2
  3
```

while in NX it is:
```
bgp confederation peers 1 2 3
```

(hint hint it sure would be nice to document this somewhere).

So for NX config_get() returns a single value '1 2 3' but for XR it returns the array ['1', '2', '3'] (hence the weird logic in the node_utils class).

Assuming I'm correct in my understanding, I would think that default_value should be `""` for NX but `[]` for XR. `kind` can be `string` for NX but not for XR (maybe?)